index.js

```diff
@@ -502,12 +535,20 @@ Router.prototype.use = function use(handler) {
  * and middleware to routes.
  *
  * @param {string} path
+ * @param {string} name (optional)
```

Comment:
Rather than making the argument's description "(optional)", can we use the JSDoc features to make it as optional? Probably a better IDE experience.